### ISSUE TITLE
Column heading in the third table of the netcdf attributes is misleading

### ISSUE THREAD

**Issue by mortenwh** [metno/py-mmd-tools#158]
A user skipped the metadata addition because the heading was "Extra Attribute". I suggest to change it to "Necessary non-ACDD attribute". Also the recommended attributes should probably be "highly recommended"..?

- [x] change column heading
- [x] remove collection from yaml file
- [x] add extra "help"/"comment" column in the tables or exchange the heading "Repetition" with "Comment/help"
- [x] Clarify use of the attributes in the comment/help column

**Comment by mortenwh** [comment]
I've tried to add the comments under each acdd field instead of under the mmd fields but only for the `metadata_identifier`. That caused `mmd_to_nc.py` to fail. I discovered that the translation of that field is only taking into account the `id` attribute in ACDD, and not `naming_authority`. @ElodieFZ - is this on purpose? Does the program split the `metadata_identifier` into `naming_authority` and `id` somewhere else?

Putting the comments under the acdd names is probably the best solution for the documentation but requires a bit of work..
